base/cvd/cuttlefish/host/libs/web/credential_source.cc

```diff
@@ -73,20 +73,22 @@ Result<std::unique_ptr<CredentialSource>> GetCredentialSourceLegacy(
   if (credential_source == "gce") {
     result = GceMetadataCredentialSource::Make(http_client);
   } else if (credential_source == "") {
-    LOG(VERBOSE) << "Probing oauth credentials at " << oauth_filepath;
     if (FileExists(oauth_filepath)) {
       std::string oauth_contents = CF_EXPECT(ReadFileContents(oauth_filepath));
       auto attempt_load = RefreshCredentialSource::FromOauth2ClientFile(
           http_client, oauth_contents);
       if (attempt_load.ok()) {
         result.reset(new RefreshCredentialSource(std::move(*attempt_load)));
+        LOG(INFO) << "\"" << oauth_filepath
+                  << "\" file found, using that as credentials";
```

Comment:
nit: same comment as below about dropping the word `file`.

Reply:
Done.